src/build/GhosttyResources.zig

```diff
@@ -16,6 +17,12 @@ pub fn init(b: *std.Build, cfg: *const Config) !GhosttyResources {
 
     // Terminfo
     terminfo: {
+        const os_tag = cfg.target.result.os.tag;
+        const terminfo_share_dir = if (os_tag == .freebsd)
```

Comment:
I don't get this part. We do have `/usr/local/share/terminfo`, but not `site-terminfo`.

Skimming through FreeBSD's `man terminfo` there seem to be no way to add terminfo data at all. So I'd rather revert these changes to simplify ghostty code and then do a full research how 3rd party terminfos should be installed.

Reply:
```
/usr/local/share/site-terminfo
/usr/local/share/site-terminfo/f
/usr/local/share/site-terminfo/f/foot
/usr/local/share/site-terminfo/f/foot-direct
```
```
cat: /usr/local/share/terminfo: No such file or directory
```

14.3-n271432-8c9ce319fef7

Reply:
```
/usr/local/share/site-terminfo/f/foot-direct was installed by package foot-1.22.3
```